instrumentation/opentelemetry-api-beta/src/main/java/io/opentelemetry/instrumentation/auto/opentelemetryapi/metrics/ApplicationDoubleCounter.java

```diff
@@ -37,6 +37,11 @@ public void add(double delta, Labels labels) {
     agentDoubleCounter.add(delta, LabelBridging.toAgent(labels));
   }
 
+  @Override
+  public void add(double v) {
```

Comment:
Don't know how the coverage of these tests is but if it's possible to add that'd be nice. When adding these bridges I needed them to be (mostly) sure my bridging is sound. Though admittedly these seem simpler than a lot of the tracing / context related stuff.

Reply:
I am not aware of any metric-related tests in our repo. Including these bridges

Reply:
See https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/master/instrumentation/opentelemetry-api-beta/src/test/groovy/MeterTest.groovy

Reply:
I was unable to understand in 2 minutes how to better do that. If this is blocker, I will have to think about it tomorrow, with fresh head

Reply:
No problem to open issue(s) for anything unresolved in this PR, better to merge it sooner to resolve #1102

Reply:
Then feel free to approve and merge if build is green :)

Reply:
I created #1105 for this